src/drivers/intel/cavs/idc.c

```diff
@@ -91,8 +91,12 @@ static bool idc_is_received(int target_core)
  */
 static bool idc_is_powered_up(int target_core)
 {
+#if CONFIG_IPC_MAJOR_4
```

Comment:
@dnikodem should this code be in the Zephyr IDC driver ? i.e. when using Zephyr all the drivers will use the Zephyr APIs and be part of upstream Zephyr ? 
@abonislawski @juimonen can you confirm the IDC is fully ready for this feature in Zephyr ?